lib/tmuxinator/project.rb

```diff
@@ -208,7 +217,12 @@ def tmux_options
     end
 
     def base_index
-      get_base_index.to_i
+      base = 0
+      if in_current_session
+        # Get the last window index + 1
+        base = 1 + `tmux list-windows | awk '{print $1}'`.split.last.to_i
```

Comment:
I think this awk logic should either be handled in Ruby code or we should add a method to the Doctor class which verifies that awk exists on the user's system.

Reply:
fair, let's me see what I can do